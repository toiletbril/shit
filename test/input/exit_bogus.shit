exit "hello world" "hello world"
