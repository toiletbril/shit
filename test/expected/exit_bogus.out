[CompoundList]
  [CompoundListCondition, None]
    [SimpleCommand "exit hello world hello world"]
[Escape Map
  
]
shit: Goodbye :c
